vic/drivers/cesm/readme.md

```diff
@@ -26,10 +25,11 @@ The CESM driver for VIC can be built in two ways.
   cd $HOME
   export VIC5TESTDIR=$HOME/vic5_inputdata
   cd $VIC5TESTDIR
-  /usr/bin/curl --header 'Host: dl-web.dropbox.com' --header 'User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10.10; rv:40.0) Gecko/20100101 Firefox/40.0' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Language: en-US,en;q=0.5' --header 'Referer: https://www.dropbox.com' --header 'Cookie: locale=en; t=_m_3OjJVG0C4_KFK1Cz1Vd_1; blid=AADO2Ivt6-gp5e0WvK8GH4c42fv31BSECHgI_d_XxhH2TQ; bjar=W3sidWlkIjogNDk1MDg2MjMsICJzZXNzX2lkIjogNzA0NjA1ODE4OTQxNjA3OTY3NzI2NjcxOTg1OTIwOTcxODcwMDEsICJleHBpcmVzIjogMTQ0MzcxNDY3OSwgInRlYW1faWQiOiAiIiwgInJvbGUiOiAicGVyc29uYWwifV0%3D' --header 'Connection: keep-alive' 'https://dl-web.dropbox.com/get/UW_RASM_shared/calibration/vic_params_wr50a_vic5.0.dev.nc?_subject_uid=49508623&w=AACXfsAfXEE1Fvjp04FKoS16Wu4jLEi-uT8Ad6dau8yYpA&dl=1' -o 'vic_params_wr50a_vic5.0.dev.nc' -L
+  /usr/bin/curl -O ftp://ftp.hydro.washington.edu/pub/jhamman/rasm_vic5_test_data/vic_params_wr50a_vic5.0.dev.nc
```

Comment:
I don't think it is necessary for now.  Soon, this dataset will be part of the RASM data and we can remove all this instruction. This is mostly so you can use the same datasets as I am.